lib/fabrik/Fabrik.js

```diff
@@ -31,12 +32,27 @@ class Fabrik {
       .then(managerConstructor => managerConstructor.load(plan));
   }
 
-  static createInstance(instance_id, service_id, plan_id) {
+  static createInstance(instance_id, service_id, plan_id, context) {
     const plan = catalog.getPlan(plan_id);
     assert.strictEqual(service_id, plan.service.id);
     return this
       .createManager(plan)
-      .then(manager => manager.createInstance(instance_id));
+      .then(manager => {
+        const instance = manager.createInstance(instance_id);
+        return Promise
+          .try(() => {
+            if (!context) {
+              return instance.platformContext;
+            } else {
+              return context;
+            }
+          })
+          .then(context => {
+            const platformManager = BasePlatformManager.getInstance(instance_id, context);
+            instance.assignPlatformManager(platformManager);
```

Comment:
Just from beautification the same block of code can look like this
```
const instance = manager.createInstance(instance_id);
return Promise.try(() => context ? context : instance.platformContext)
            .then(context => instance.assignPlatformManager(BasePlatformManager.getInstance(instance_id, context))
            .return(instance);
```

Reply:
Done. Okay with return part. But Breaking in two lines adds more ease in readability. that's why put. But made changes as per suggestion.